codespell_lib/data/dictionary.txt

```diff
@@ -11247,8 +11250,8 @@ cought->caught, cough, fought,
 coul->could
 could'nt->couldn't
 could't->couldn't
-couldent->couldn't
 coulden`t->couldn't
```

Comment:
```suggestion
coulden`t->couldn't
coulden't->couldn't
```

Reply:
Dictionary changes seem outside of the scope of this PR. I can introduce this change if needed, but I would prefer to keep this PR tightly scoped.